mock-abis-default.properties

```diff
@@ -8,8 +8,14 @@ management.endpoint.health.show-details=always
 management.endpoits.web.exposure.include=info,health,refresh
 server.port=8081
 abis.return.duplicate=false
-secret_url=${mosip.kernel.authmanager.url}/v1/authmanager/authenticate/clientidsecretkey
-secret_url.clientnId=mosip-regproc-client
-secret_url.id=string
-secret_url.secretKey=${mosip.regproc.client.secret}
-secret_url.appId=regproc
+#iam adapter
+auth.server.admin.issuer.uri=${keycloak.external.url}/auth/realms/
+mosip.iam.adapter.appid=regproc
+mosip.iam.adapter.clientid=mosip-regproc-client
+mosip.iam.adapter.clientsecret=${mosip.regproc.client.secret}
+mosip.kernel.auth.adapter.ssl-bypass=true
+mosip.kernel.auth.appid-realm-map={regproc:'mosip'}
+mosip.kernel.auth.appids.realm.map={regproc:'mosip'}
```

Comment:
this and above looks like duplicate with minor change in name, why we need this ?

Reply:
Both are not duplicate .because same I have seen in https://github.com/mosip/mosip-config/blob/develop/application-default.properties
As mock abis is not using from application-default so added separately for this